src/main/perl/RuleBasedEditor.pm

```diff
@@ -568,13 +606,20 @@ sub _buildLinePattern
     }
     if (defined($config_value)) {
         $self->debug(2, "$function_name: configuration value '$config_value' will be added to the pattern");
+        # config_value: scape characters with a specific meaning in the regexp
         $config_value =~ s/\\/\\\\/g;
-        $config_value =~ s/([\-\+\?\.\*\[\]()\^\$])/\\$1/g;
+        $config_value =~ s/([\-\+\?\.\*\[\]()\^\$\{\}])/\\$1/g;
         $config_value =~ s/\s+/\\s+/g;
     } else {
         $config_value = "";
     }
 
+    # config_param: escape characters with a specific meaning in the regexp.
```

Comment:
if this is the same escapiong as above, put it in a (anonymous?) sub instead of copy pasting